packages/collector/Collector_Db.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+function collector_esc_sql_identifier($tableName) {
+	return str_replace('`', '``', esc_sql($tableName));
+}
+
+function collector_dump_db($zip)
+{
+	$tables   = collector_get_db_tables();
+	$sql_dump = '';
+
+	$sql_dump .= sprintf("-- %s\n", json_encode(['SECTION START' => 'SCHEMA']));
+
+	foreach($tables as $table)
+	{
+		$sql_dump .= sprintf("-- %s\n", json_encode(['ACTION' => 'DROP', 'TABLE' => $table]));
+		$sql_dump .= sprintf("DROP TABLE IF EXISTS `%s`;\n", collector_esc_sql_identifier($table));
```

Comment:
Is there any `$wpdb` escaping utility better suited for the job? I imagine this could fail in a few ways when backslashes are involved.

Reply:
I updated this to use `esc_sql`